client/app/src/screens/Results.js

```diff
@@ -1,12 +1,244 @@
 import React, { Component } from 'react';
-import { Text, View } from 'react-native';
+import { Platform, Text, TouchableOpacity, View } from 'react-native';
 import EStyleSheet from 'react-native-extended-stylesheet';
+import SideSwipe from 'react-native-sideswipe';
+import { AreaChart, BarChart, LineChart, StackedAreaChart } from 'react-native-svg-charts';
+import { curveNatural } from 'd3-shape';
+import { scaleTime } from 'd3-scale';
+import Constants from 'expo-constants';
+
+import { store } from '../state';
+import Layout from '../constants/Layout';
+import COLORS, { swatchColorInfo } from '../constants/Colors';
+import { CHOICE_COLOR, CHOICE_CHILLS } from '../constants/Choices';
+
+const COLOR_KEYS = Object.keys(swatchColorInfo);
+const COLOR_EMPTY = Object.fromEntries(COLOR_KEYS.map(color => [color, 0]));
+const CARD_MARGIN = 0.05 * Layout.window.width;
```

Comment:
Another front end ignorance question: how come none of the widths/heights here take up the full 100%?

Reply:
Aesthetic choice. I want them to look like cards. If you look at the screenshots and then imagine (or just try it) the margin being gone, I think it would be less intuitive that these are swipeable elements.